.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/Azure/sonic-swss-common
 [submodule "sonic-linux-kernel"]
 	path = src/sonic-linux-kernel
-	url = https://github.com/Azure/sonic-linux-kernel
+	url = https://github.com/InterfaceMasters/sonic-linux-kernel.git
```

Comment:
i think we have a section in our 2624 dev guide and we note repos from mainstream for sonic and repos hosted on IMT site... can you pls check and update?

Reply:
I will update guide when this PR will be merged